RUFAS/simulation_engine.py

```diff
@@ -177,20 +184,40 @@ def _annual_simulation(self) -> None:
 
         self._run_post_annual_routines()
 
+    def annual_reset(self):
+        """
+        Description:
+            Resets all annual variables that require reset
+        """
+        self.field_manager.annual_update_routine()
+
+    def annual_mass_balance(self, time):
+        pass
+
     def _initialize_simulation(self) -> None:
         """
         Instantiates the simulation object by requesting data from the Input Manager.
         """
 
-        data_config = im.get_data("config")
+        data_config: Dict[str, Any] = im.get_data("config")
         data_weather = im.get_data("weather")
```

Comment:
```suggestion
        config_data: Dict[str, Any] = im.get_data("config")
        weather_data = im.get_data("weather")
```
Not an important change, but I think this reads better